app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/TransformAdd.tsx

```diff
@@ -86,7 +86,7 @@ export default function TransformAddButton({
       directive: directiveText,
     };
 
-    if (directive === 'tink') {
+    if (directive.toLowerCase() === 'tink') {
       fullDirective = addTinkToTransforms(transformInfo);
     } else if (directive === 'Rename') {
```

Comment:
Do we also need to compare lower case for `Rename`?

Reply:
I think we should change the call to `handleMenuClick` to pass the directive name in, instead of relying what's in the UI.

Reply:
the text that shows up on the page is `Rename` and for tink its `TINK`

Reply:
i didn't see nick's comment - yeah i see what you mean